.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2.1
+
+orbs:
+    browser-tools: circleci/browser-tools@0.1.0
+
+executors:
+  encoded-executor:
+    docker:
+      - image: circleci/buildpack-deps:trusty
```

Comment:
just to check, I think this is 14.04 what we run in production... it's hard to say because the ami cloud-config uses is pretty old (again).  @clitton?

Reply:
I think trusty is 14.04.X  And is updated the x as needed.  Now its 14.04.06.